content/blog/2016/2016-08-25-sauce-pipeline.adoc

```diff
@@ -0,0 +1,262 @@
+---
+layout: post
+title: "Running Sauce On-Demand in Pipeline"
```

Comment:
This title lacks a little giddy-up in my opinion, work for you if you recognize what "sauce on-demand" is but I don't think it resonates if you're not already familiar with their service.

Here are some suggestions:

* "Browser-testing with Sauce On-Demand and Pipeline"
* "Web QA with Sauce On-Demand and Pipeline"
* "Web Acceptance Testing with Pipeline"

Or some combination thereof, I'm not married to any one in particular, just brainstorming